dev/sbt-plugin/src/sbt-test/sbt-plugin/akka-persistence-typed-migration-scala/shopping-cart-api/src/main/scala/com/example/shoppingcart/api/ShoppingCartService.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example.shoppingcart.api
+
+import java.time.Instant
+
+import akka.{Done, NotUsed}
+import com.lightbend.lagom.scaladsl.api.transport.Method
+import com.lightbend.lagom.scaladsl.api.{Service, ServiceCall}
+import play.api.libs.json.{Format, Json}
+
+object ShoppingCartService  {
+  val TOPIC_NAME = "shopping-cart"
+}
+
+/**
+  * The ShoppingCart service interface.
+  * <p>
+  * This describes everything that Lagom needs to know about how to serve and
+  * consume the ShoppingCartService.
+  */
+trait ShoppingCartService extends Service {
+
+  /**
+    * Get a shopping cart.
+    *
+    * Example: curl http://localhost:9000/shoppingcart/123
+    */
+  def get(id: String): ServiceCall[NotUsed, String]
+
+  /**
+   * Get a shopping cart report (view model).
+   *
+   * Example: curl http://localhost:9000/shoppingcart/123/report
+   */
+  def getReport(id: String): ServiceCall[NotUsed, String]
+
+  /**
+    * Update an items quantity in the shopping cart.
+    *
+    * Example: curl -H "Content-Type: application/json" -X POST -d '{"productId": 456, "quantity": 2}' http://localhost:9000/shoppingcart/123
+    */
+  def updateItem(id: String, productId: String, qty: Int): ServiceCall[NotUsed, Done]
+
+  /**
+    * Checkout the shopping cart.
+    *
+    * Example: curl -X POST http://localhost:9000/shoppingcart/123/checkout
+    */
+  def checkout(id: String): ServiceCall[NotUsed, Done]
```

Comment:
A few of the comments are now out of date (no longer `POST`s). I often leave these comments out in `scripted` tests since the `test` script already demonstrates how to use the endpoint.